Cache/Requests.lean

```diff
@@ -27,18 +48,22 @@ def getToken : IO String := do
 
 open System (FilePath)
 
+/-- The full name of the main Mathlib GitHub repository. -/
+def MATHLIBREPO := "leanprover-community/mathlib4"
+
 /--
 Given a file name like `"1234.tar.gz"`, makes the URL to that file on the server.
 
 The `f/` prefix means that it's a common file for caching.
 -/
-def mkFileURL (URL fileName : String) : String :=
-  s!"{URL}/f/{fileName}"
+def mkFileURL (repo URL fileName : String) : String :=
+  let pre := if repo == MATHLIBREPO then "" else s!"{repo}/"
```

Comment:
Why not just keep the prefix either way?

A one off cost of invalidating the entire cache is fine, in exchange for some simplicity/uniformity.

Reply:
Backwards compatibility was easy enough to support, so it seemed worth doing. If we are okay with invalidating the whole cache, perhaps we should switch to a different prefix (other than `f/`) to make everything under that prefix be grouped by repository.

Reply:
Okay, let's leave as is for now. Better to invalidate everything in a separate PR if we're going to do that.